src/main/webapp/i18n/en/tutorialGroupExport.json

```diff
@@ -0,0 +1,13 @@
+{
+    "artemisApp": {
+        "tutorialGroupExportDialog": {
+            "buttonHeader": "Export Tutorial Groups to CSV",
```

Comment:
Rename to "Export Tutorial Groups" as it aligns better with "Import Tutorial Groups".

Reply:
👍